scripts.js

```diff
@@ -0,0 +1,16 @@
+$(function () {
```

Comment:
What is the purpose of this file, why is it in the root of the project? And what is the file `public/scripts.js`?